R/eval.R

```diff
@@ -1,19 +1,18 @@
-#' Evaluate R code and mask the output by a prefix
+#' Insert output to source code
 #'
-#' This function is designed to insert the output of each chunk of R code into
-#' the source code without really breaking the source code, since the output is
-#' masked in comments.
-#' @param source the input filename (by default the clipboard; see
-#'   \code{\link{tidy_source}()})
-#' @param ... other arguments passed to \code{\link{tidy_source}()}
-#' @param file the file to write by \code{\link{cat}()}; by default the output
-#'   is printed on screen
-#' @param prefix the prefix to mask the output
-#' @param envir the environment in which to evaluate the code (by default the
-#'   parent environment; if we do not want to mess up with the parent
-#'   environment, we can set \code{envir = NULL} or \code{envir = new.env()})
+#' Evaluates R code by chunks, then inserts the output to each chunk.
```

Comment:
I wouldn't use the third-person singular:

```suggestion
#' Evaluate R code by chunks, then insert the output to each chunk.
```

Reply:
Should all descriptions follow this rule?

Reply:
Yes.